ui/pages/create-account/connect-hardware/select-hardware.js

```diff
@@ -296,6 +319,67 @@ export default class SelectHardware extends Component {
     );
   }
 
+  renderQRHardwareWalletSteps() {
+    const steps = [];
+    steps.push({
```

Comment:
Since none of these steps are conditionally added to the array, can we just do:

```
const steps = [
    { 
        title: this.context.t('QRHardwareWalletSteps1Title'),
        message: this.context.t('QRHardwareWalletSteps1Description') 
    },
    // more items ....
];
```

Reply:
Sure

Reply:
@darkwing 😄 ， updated the code style.